compiler/code-gen/declarations.cpp

```diff
@@ -906,10 +906,6 @@ void ClassDeclaration::compile_accept_json_visitor(CodeGenerator &W, ClassPtr kl
 }
 
 void ClassDeclaration::compile_accept_visitor_methods(CodeGenerator &W, ClassPtr klass) {
-  if (G->is_output_mode_k2()) {
-    // The current version of runtime-light does not support visitors
-    return;
-  }
   bool need_generic_accept =
```

Comment:
Let's add condition to skip generation of visitors for `instance_cache` and `memory_estimate` cases. For now it generate c++ compile errors if I try code like `instance_cache_store("key", $value);`